web/src/lib/components/album-page/album-options.svelte

```diff
@@ -77,22 +113,42 @@
           </div>
           <div>{$t('invite_people')}</div>
         </button>
+        
+        {#if user} <!-- Use user prop instead of $user -->
```

Comment:
I don't think you need that check. It's already checked here https://github.com/immich-app/immich/blob/6c7d51da34ca20011062efcf2c67ada5b2d0e30f/web/src/routes/(user)/albums/%5BalbumId%3Did%5D/%5B%5Bphotos%3Dphotos%5D%5D/%5B%5BassetId%3Did%5D%5D/%2Bpage.svelte#L738